operator.yaml

```diff
@@ -875,6 +881,9 @@ spec:
     - jsonPath: .status.address.url
       name: Address
       type: string
+    - jsonPath: .status.endpoints[?(@.portName == "NodePort")].port
+      name: Port
+      type: string
```

Comment:
I think `endpoint` should not be an array. Why it is? `Endpoint` can be the exposed URI and `Address` to respect Knative interface? Address can be internal, Endpoint can be external.

Reply:
I used an array since in the initial design notes you introduced the `endpoints` as a plural entity and I  was thinking that it is due to the fact that with multiple pods there will be multiple endpoints if the kube scheduler will spread the pods in different nodes (in a production scenario it could happen quite often).
Anyway we can refine the design accordingly to your concern if we think that the reported use case could be managed using a different approach.

Reply:
We problably get inspirated by Kubernetes for the `Endpoints` as a plural entity https://kubernetes.io/docs/reference/kubernetes-api/service-resources/endpoints-v1/ but if this doesn't fit our needs we can change it

Reply:
I simplified the things based on this discussion and in particular I removed the array of `Endpoints`.
Now there is only one `Endpoint` that we are using to old the external URL the workflow will be available to.